runtime/doc/helphelp.txt

```diff
@@ -246,6 +246,54 @@ command: >
 			To rebuild the help tags in the runtime directory
 			(requires write permission there): >
 				:helptags $VIMRUNTIME/doc
+<
+								*help-TOC*
+
+If you want to access an interactive table of contents, from any position in
+the file, you can use the helptoc plugin.  Load the plugin with: >
+
+    packadd helptoc
+
+Then you can use the `:HelpToc` command to open a popup menu.
+The latter supports the following normal commands: >
+
+	key   | effect
+	------+---------------------------------------------------------
+	j     | select next entry
+	k     | select previous entry
+	J     | same as j, and jump to corresponding line in main buffer
+	K     | same as k, and jump to corresponding line in main buffer
+	c     | select nearest entry from cursor position in main buffer
+	g     | select first entry
+	G     | select last entry
+	H     | collapse one level
+	L     | expand one level
+	p     | print current entry on command-line
+	      |
+	P     | same as p but automatically, whenever selection changes;
+	      | press multiple times to toggle feature on/off
+	      |
+	q     | quit menu
+	z     | redraw menu with current entry at center
+	<C-D> | scroll down half a page
+	<C-U> | scroll up half a page
```

Comment:
Can you add the support for using the PageUp and PageDown keys to scroll the menu?
Also, Home key to select the first entry and End to select the last entry?